docs/jwst/introduction.rst

```diff
@@ -26,7 +26,7 @@ and :ref:`stage2-spectroscopic-flow`.
 Stage 3 processing consists of routines that combine the data from
 multiple exposures, for all observing modes.
 Details of stage 3 imaging processing are at
-:ref:`stage3-imaging-flow`.
+:ref:`stage3-imaging-flow-blahblah`.
```

Comment:
Why is this reference being changed to include "blahblah"? I don't think that's a reference to an actual doc anywhere.

Reply:
This is just to see if `build_docs` (or `build_sphinx`) is doing what it is supposed to do, i.e., to catch any Sphinx warnings. This is because I don't see you have the `-w` option as `astropy` has, so not sure if that is oversight or not, as you don't use `astropy-helpers`. I expect this commit to cause `build_docs` to fail. If it does not, you have a problem. This particular change should not be merged.

Reply:
https://travis-ci.org/STScI-JWST/jwst/jobs/293305788 did not fail despite "544 warnings" stated in its log. That is a bit concerning.